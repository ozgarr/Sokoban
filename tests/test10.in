---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------

r2
r8
q4
q8
q6
q2
f6
l8
m4
m4
p6
a8
p6
o4
o4
l2
h6
o8
m6
j8
i6
c2
d2
h6
i4
k8
n8
e6
g2
i2
n8
e2
g2
r4
n4
o8
m6
i8
g6
d2
e4
i8
m8
o6
n8
n6
h4
e2
f4
o6
l8
m8
m6
n2
n4
n6
b6
a6
a6
a6
a4
a4
a4
a4
a4
a4
a4
a4
o2
o2
o2
o2
o2
o2
o8
o6
o6
o6
o6
o6
o6
o6
o6
o6
o6
r8
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
.
