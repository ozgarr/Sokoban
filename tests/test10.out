---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----n---h-----#----r---
-----mlkji-----#--------
-@-------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----nl--h-----#----r---
-----m@kji-----#--------
---------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----nl--h-----#----r---
----m@-kji-----#--------
---------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----p---f-----#--------
-----o-q-g-----#--------
-----nl--h-----#----r---
---m@--kji-----#--------
---------------#--------
---------------#--------
---------------#--------
-----abcde-----#--------
-----@p--f-----#--------
-----o-q-g-----#--------
-----nl--h-----#----r---
---m---kji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
-----@bcde-----#--------
------p--f-----#--------
-----o-q-g-----#--------
-----nl--h-----#----r---
---m---kji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
------@p-f-----#--------
-----o-q-g-----#--------
-----nl--h-----#----r---
---m---kji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
-------p-f-----#--------
----o@-q-g-----#--------
-----nl--h-----#----r---
---m---kji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
-------p-f-----#--------
---o@--q-g-----#--------
-----nl--h-----#----r---
---m---kji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
-------p-f-----#--------
---o---q-g-----#--------
-----n@--h-----#----r---
---m--lkji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
-------p-f-----#--------
---o---q-g-----#--------
-----n---@h----#----r---
---m--lkji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
---o---p-f-----#--------
---@---q-g-----#--------
-----n----h----#----r---
---m--lkji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
---o---p-f-----#--------
-------q-g-----#--------
-----n----h----#----r---
---@m-lkji-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
---o---p-f-----#--------
-------q-g-----#--------
-----n--j-h----#----r---
----m-lk@i-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
---o---p-f-----#--------
-------q-g-----#--------
-----n--j-h----#----r---
----m-lk-@i----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bcde-----#--------
---o---p-f-----#--------
-------q-g-----#--------
-----n--j-h----#----r---
----m-lk-@i----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bc@e-----#--------
---o---pdf-----#--------
-------q-g-----#--------
-----n--j-h----#----r---
----m-lk--i----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bc-e-----#--------
---o---pdf-----#--------
-------q-g-----#--------
-----n--j-@h---#----r---
----m-lk--i----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bc-e-----#--------
---o---pdf-----#--------
-------q-g-----#--------
-----n--j--h---#----r---
----m-lk-i@----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bc-e-----#--------
---o---pdf-----#--------
-------q-g-----#--------
-----n-kj--h---#----r---
----m-l@-i-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bc-e-----#--------
---o---pdf-----#--------
-----n-q-g-----#--------
-----@-kj--h---#----r---
----m-l--i-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bc-@e----#--------
---o---pdf-----#--------
-----n-q-g-----#--------
-------kj--h---#----r---
----m-l--i-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bc-@e----#--------
---o---pdf-----#--------
-----n-q-g-----#--------
-------kj--h---#----r---
----m-l--i-----#--------
---------------#--------
---------------#--------
-----a---------#--------
------bc--e----#--------
---o---pdf-----#--------
-----n-q-g-----#--------
-------kj--h---#----r---
----m-l--@-----#--------
---------i-----#--------
---------------#--------
-----a---------#--------
------bc--e----#--------
---o-n-pdf-----#--------
-----@-q-g-----#--------
-------kj--h---#----r---
----m-l--------#--------
---------i-----#--------
---------------#--------
-----a---------#--------
------bc--@----#--------
---o-n-pdfe----#--------
-------q-g-----#--------
-------kj--h---#----r---
----m-l--------#--------
---------i-----#--------
---------------#--------
-----a---------#--------
------bc--@----#--------
---o-n-pdfe----#--------
-------q-g-----#--------
-------kj--h---#----r---
----m-l--------#--------
---------i-----#--------
---------------#--------
-----a---------#--------
------bc--@----#--------
---o-n-pdfe----#--------
-------q-g-----#--------
-------kj--h---#----r---
----m-l--------#--------
---------i-----#--------
---------------#--------
-----a---------#--------
------bc-------#--------
---on@-pdfe----#--------
-------q-g-----#--------
-------kj--h---#----r---
----m-l--------#--------
---------i-----#--------
---------------#--------
-----a---------#--------
---o--bc-------#--------
---@n--pdfe----#--------
-------q-g-----#--------
-------kj--h---#----r---
----m-l--------#--------
---------i-----#--------
---------------#--------
-----a---------#--------
---o--bc-------#--------
----n--pdfe----#--------
-------q-g-----#--------
-------kj--h---#----r---
----@ml--------#--------
---------i-----#--------
---------------#--------
-----a---------#--------
---o--bc-------#--------
----n--pdfe----#--------
-------q-g-----#--------
-------kj--h---#----r---
-----ml--i-----#--------
---------@-----#--------
---------------#--------
-----a---------#--------
---o--bc-------#--------
----n--pdfe----#--------
-------q-g-----#--------
-------kj--h---#----r---
-----ml--i-----#--------
---------@-----#--------
---------------#--------
-----a---------#--------
---o--bc-------#--------
----n--p@fe----#--------
-------qdg-----#--------
-------kj--h---#----r---
-----ml--i-----#--------
---------------#--------
---------------#--------
-----a---------#--------
---o--bc-------#--------
----n--p@fe----#--------
-------qdg-----#--------
-------kj--h---#----r---
-----ml--i-----#--------
---------------#--------
---------------#--------
-----a---------#--------
---o--bc-------#--------
----n--p-fe----#--------
-------qdg-----#--------
-------kji-h---#----r---
-----ml--@-----#--------
---------------#--------
---------------#--------
-----a---------#--------
---o--bc-------#--------
----n--p-fe----#--------
-------qdg-----#--------
-----m-kji-h---#----r---
-----@l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
---@o-bc-------#--------
----n--p-fe----#--------
-------qdg-----#--------
-----m-kji-h---#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
---@o-bc-------#--------
----n--p-fe----#--------
-------qdg-----#--------
-----m-kji-h---#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----o-bc-------#--------
----@n-p-fe----#--------
-------qdg-----#--------
-----m-kji-h---#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----o-bc-------#--------
-----n-p-fe----#--------
-------qdg-----#--------
-----m-kjih@---#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----o-bc-------#--------
-----n-p-f@----#--------
-------qdge----#--------
-----m-kjih----#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----o-bc-------#--------
-----n-pf@-----#--------
-------qdge----#--------
-----m-kjih----#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----@obc-------#--------
-----n-pf------#--------
-------qdge----#--------
-----m-kjih----#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----n-pf------#--------
-------qdge----#--------
-----mlkjih----#----r---
------@--------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----n-pf------#--------
-----m-qdge----#--------
-----@lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----n-pf------#--------
-----@mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----n-pf------#--------
-----@mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----n-pf------#--------
-----@mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----@npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----@npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----@a--------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
------@a-------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-------@a------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-------a@------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
------a@-------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a@--------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
----a@---------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
---a@----------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
--a@-----------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-a@------------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
a@-------------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
a--------------#--------
-----@bc-------#--------
-----onpf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
a--------------#--------
------bc-------#--------
-----@npf------#--------
-----omqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
-----@mqdge----#--------
-----olkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
-----@lkjih----#----r---
-----o---------#--------
---------------#--------
---------------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
-----@---------#--------
-----o---------#--------
---------------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
-----@---------#--------
-----o---------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
-----@---------#--------
-----o---------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-----@o--------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
------@o-------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-------@o------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
--------@o-----#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------@o----#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
----------@o---#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-----------@o--#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
------------@o-#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-------------@o#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-------------@o#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-------------@o#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
------------@o-#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-----------@o--#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
----------@o---#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------@o----#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
--------@o-----#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-------@o------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
------@o-------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
-----@o--------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
-----@---------#--------
-----o---------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
-----@---------#--------
-----o---------#--------
---------------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
------mqdge----#--------
-----@lkjih----#----r---
-----o---------#--------
---------------#--------
---------------#--------
a--------------#--------
------bc-------#--------
------npf------#--------
-----@mqdge----#--------
-----olkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
a--------------#--------
------bc-------#--------
-----@npf------#--------
-----omqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
a--------------#--------
-----@bc-------#--------
-----onpf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
a@-------------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-a@------------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
--a@-----------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
---a@----------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
----a@---------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a@--------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
------a@-------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-------a@------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-------@a------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
------@a-------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----@a--------#--------
-----obc-------#--------
------npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----@npf------#--------
------mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----n-pf------#--------
-----@mqdge----#--------
------lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----n-pf------#--------
-----m-qdge----#--------
-----@lkjih----#----r---
---------------#--------
---------------#--------
---------------#--------
-----a---------#--------
-----obc-------#--------
-----n-pf------#--------
-------qdge----#--------
-----mlkjih----#----r---
------@--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----@obc-------#--------
-----n-pf------#--------
-------qdge----#--------
-----m-kjih----#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----o-bc-------#--------
-----n-pf@-----#--------
-------qdge----#--------
-----m-kjih----#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----o-bc-------#--------
-----n-p-f@----#--------
-------qdge----#--------
-----m-kjih----#----r---
------l--------#--------
---------------#--------
---------------#--------
-----a---------#--------
----o-bc-------#--------
-----n-p-fe----#--------
-------qdg-----#--------
-----m-kjih@---#----r---
------l--------#--------
---------------#--------
---------------#--------
